src/app/Http/routes.php

```diff
@@ -117,6 +117,7 @@
 
             Route::group(['middleware' => ['auth', 'banned', 'verified', 'nophonenumber']], function () {
                 Route::get('/account', 'AccountController@index');
+                Route::get('/account/ticket/pdf/{ticketId}', 'AccountController@pdfTicket');
```

Comment:
This should not be `/account/ticket/pdf/{ticketId}` but `/events/participants/{participant}` and linking to a function on Events/ParticipantsController.php which then calls the pdfTicket function that should be defined in the EventParticipant itself

Reply:
Can we make this `/tickets/pdf/{participant}`? I feel like this makes more logical sense, since related functions (namely `retrieve`) are referenced by the QR code and the `ParticipantsController` pretty much only deals with gifting.

Reply:
Added route